### PR TITLE
Added enumerated value (TURRET_FRAME and title case version) to coord…

### PR DESCRIPTION
**Pull request by mit3ch**
…inate_space_frame_type to support Mars2020.

## Summary

Enter summary here

## Test Data and/or Report

Enter info here

## Related Issues

Enter related issues here

### CODE REVIEW

File: src/PDS4_GEOM_IngestLDD.xml [hunk 1]
```diff
@@ -2733,7 +2736,7 @@
             
             <DD_Permissible_Value>
              <value>ROVER_FRAME</value>
-             <value_meaning>Frame that is attached to the rover and moves with the rover (in both position and orientation). Often synonymous with ROVER_NAV_FRAME.</value_meaning>
```
Comment (jordanpadams) [inline src/PDS4_GEOM_IngestLDD.xml:2736]: @mit3ch was the removal of these definitions intentional? it looks to have occurred for a few enumerated values

**Review by jordanpadams — CHANGES_REQUESTED** [review]
@mit3ch was the removal of these definitions intentional? it looks to have occurred for a few enumerated values

File: src/PDS4_GEOM_IngestLDD.xml (hunk 1 above)
Comment (thareUSGS) [reply src/PDS4_GEOM_IngestLDD.xml:2736]: @mit3ch @jordanpadams I noticed that too and kept them in from my quick enumeration updates. I hope that was okay. If not you can dismiss my dev-mkg-002 recent updates and additions. Looks like dev-mkg-002 is building now with "main".

**Comment by rgdeen** [comment]
Coming to the party late but... I'm not aware of anything that should have been removed... least of all ROVER_FRAME!!  So @thareUSGS thanks for keeping them in, if I interpreted the above properly.

**Pull request merged**